README.md

```diff
@@ -48,7 +49,7 @@ After installing the `auth0-lock` module, you'll need to bundle it up along with
 If you are targeting mobile audiences, we recommended that you add:
 
 ```html
-<meta name="viewport" content="width=device-width, initial-scale=1"/>
+<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1"/>
```

Comment:
that's a limitation we can't have. does the PR depend on this setting?

Reply:
How about we keep both?
-
If you are targeting mobile audiences, we recommended that you add:

```html
<meta name="viewport" content="width=device-width, initial-scale=1"/>
```
and for better experience on iOS devices add:

```html
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1"/>
```

Reply:
It's generally recommended that you don't prevent scaling, as that's annoying and potentially an accessibility problem, so I wouldn't change this at all

Reply:
Thanks for guiding. I have worked on both the changes you asked for.